integration-tests/tests/run-analysis-azure.ts

```diff
@@ -101,9 +100,11 @@ const testRunAnalysisAzure = _.flowRight(
   await deleteRuntimesV2({ page, billingProject, workspaceName });
 });
 
+// Run this test manually against staging when needed. Note the very long timeout; Azure VMs can take 20 minutes to create
 registerTest({
   name: 'run-analysis-azure',
   fn: testRunAnalysisAzure,
-  targetEnvironments: ['dev'],
-  timeout: Millis.ofMinutes(20),
+  // targetEnvironments: ['dev'],
+  targetEnvironments: ['dev', 'staging'], // uncomment for manually triggered runs against staging - DO NOT COMMIT
```

Comment:
I hadn't intended to - I didn't mean to commit this >_<

Reply:
That's because I was reviewing individual commits to avoid having to re-review the entire PR. One thing that is really helpful on PRs that have multiple reviews is if you squash all commits since the last review so there is only 1 commit that the reviewer needs to look at. But I have approved the PR now, so no need at this point.

Is there a plan to enable the test again after this merges?

Reply:
I think it was more than that; I was seeing the out-of-date code on the main /code page. I think there was a Github delay.

IA is planning to run the test manually at staging release time for now, since it's now up to 25min long. If create time decreases enough we'll probably want to reenable it for automatic runs